File: ytube-extractor/src/main/java/com/raka/ytube_extractor/models/youtube/playerResponse/RunsItem.java
package com.raka.ytube_extractor.models.youtube.playerResponse;

import java.io.Serializable;

public class RunsItem implements Serializable {
    private String text;

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    @Override
    public String toString() {
        return
                "RunsItem{" +
                        "text = '" + text + '\'' +
                        "}";
    }
}

File: ytube-extractor/src/main/java/com/raka/ytube_extractor/models/newModels/RunsItem.kt
package com.raka.ytube_extractor.models.newModels

import com.google.gson.annotations.SerializedName

class RunsItem {
    @SerializedName("text")
    var text: String? = null
    override fun toString(): String {
        return "RunsItem{" +
                "text = '" + text + '\'' +
                "}"
    }
}